core/src/main/scala/org/apache/spark/deploy/worker/ExternalShuffleService.scala

```diff
@@ -18,12 +18,15 @@
 package org.apache.spark.deploy.worker
 
 import org.apache.spark.{Logging, SparkConf, SecurityManager}
+import org.apache.spark.util.Utils
 import org.apache.spark.network.TransportContext
 import org.apache.spark.network.netty.SparkTransportConf
 import org.apache.spark.network.sasl.SaslRpcHandler
 import org.apache.spark.network.server.TransportServer
 import org.apache.spark.network.shuffle.ExternalShuffleBlockHandler
 
+import java.util.concurrent.CountDownLatch
```

Comment:
this needs to be up there above the spark imports (see https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide)